sn_networking/src/cmd.rs

```diff
@@ -181,6 +181,39 @@ pub enum SwarmCmd {
     FetchCompleted(RecordKey),
 }
 
+impl SwarmCmd {
+    /// Check if the command can be dropped safely in times of high load.
+    pub(crate) fn can_be_dropped_safely(&self) -> bool {
+        match self {
+            SwarmCmd::Dial { .. } => true,
+            SwarmCmd::DialWithOpts { .. } => true,
+            SwarmCmd::GetAllLocalPeers { .. } => false,
+            SwarmCmd::GetKBuckets { .. } => true,
+            SwarmCmd::GetClosestKLocalPeers { .. } => false,
+            SwarmCmd::GetClosestPeersToAddressFromNetwork { .. } => true,
+            SwarmCmd::GetCloseGroupLocalPeers { .. } => false,
+            SwarmCmd::GetSwarmLocalState { .. } => false,
+            SwarmCmd::SendRequest { .. } => true,
+            SwarmCmd::SendResponse { .. } => true,
+            SwarmCmd::RecordStoreHasKey { .. } => true,
```

Comment:
This is used during put_validation, we'd want this in probably as we seem to prioritize PutRecord commands here.